shared/AppInsightsCore/src/JavaScriptSDK/SenderPostManager.ts

```diff
@@ -379,8 +380,7 @@ export class SenderPostManager {
                 }
 
 
-                if (_sendCredentials && _isOneDs) {
-                    // for 1ds, Don't send credentials when URL is file://
+                if (_sendCredentials) {  // fetch request credentials
                     init.credentials = "include";
```

Comment:
This is also not exactly what is being asked.
the `credentials` value for fetch has 3 diffrent values https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch, the ask is to set this to `omit`

Reply:
yes, fetch has three and xhr only have true and false. I will combine them into one config.